scripts/microbe_stage/species_system.as

```diff
@@ -327,8 +359,14 @@ const auto MIN_BACTERIA = 2;
 //! Updates the species's population and creates new ones. And keeps track of Species objects
 class SpeciesSystem : ScriptSystem{
 
+	int currentBacteriaAmount;
+	int currentEukaryoteAmount;
+
     void Init(GameWorld@ w){
 
+		currentBacteriaAmount=0;
```

Comment:
To match existing style this initialization should be in the class (on line 362) and the variables should be moved to be after the methods where the other member variables are.

Reply:
but this variable is seperate from any individual species

Reply:
actually i get what you mean nevermind